service/lib/dinstaller/manager.rb

```diff
@@ -100,19 +74,27 @@ def options
     # * Simplified storage probing
     #
     # The initialization of these subsystems should probably live in a different place.
-    #
-    # @return [Boolean] true if the probing process ended successfully; false otherwise.
     def probe
-      change_status(InstallerStatus::PROBING)
-      probe_languages
-      probe_storage
-      @software.probe
-      true
-    rescue StandardError => e
-      logger.error "Probing error: #{e.inspect}"
-      false
-    ensure
-      change_status(InstallerStatus::IDLE)
+      Thread.new do
+        sleep(1) # do sleep to ensure that dbus service is already attached
```

Comment:
I guess this would not be needed if probing is launched by a client calling to `Manager#probe` dbus method, right?

Reply:
yeap, but my goal is to get to probed status as soon as possible. That is why I do it when starting service.